lib/split/dashboard/public/dashboard.js

```diff
@@ -22,3 +22,13 @@ function confirmReopen() {
   var agree = confirm("This will reopen the experiment. Are you sure?");
   return agree ? true : false;
 }
+
+function confirmEnableCohorting(){
+  var agree = confirm("This will enable the cohorting of the experiment. Are you sure?");
+  return agree ? true : false;
+}
+
+function confirmDisableCohorting(){
+  var agree = confirm("This will disable the cohorting of the experiment. Note: Existing participants will continue to receive their alternative and may continue to convert. Are you sure?");
+  return agree ? true : false;
+}
```

Comment:
Again already added and is in our repo: https://github.com/clio/split/blob/master/lib/split/dashboard/public/dashboard.js